scripts/tabulate_causative_ranks.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+import datetime
+import logging
+
+import click
+import coloredlogs
+import yaml
+
+# General, logging
+from scout import __version__
+from scout.adapter.client import get_connection
+
+# Adapter stuff
+from scout.adapter.mongo import MongoAdapter
+from scout.constants import CLINICAL_FILTER_BASE, CLINICAL_FILTER_BASE_SV
+
+# Controller
+from scout.server.blueprints.institutes.controllers import decorate_institute_variant
+
+try:
+    from scoutconfig import *
+except ImportError:
+    pass
+
+LOG_LEVELS = ["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"]
+LOG = logging.getLogger(__name__)
+
+
+@click.command()
+@click.option(
+    "--loglevel",
+    default="INFO",
+    type=click.Choice(LOG_LEVELS),
+    help="Set the level of log output.",
+    show_default=True,
+)
+@click.option("-db", "--mongodb", help="Name of mongo database [scout]")
+@click.option("-u", "--username")
+@click.option("-p", "--password")
+@click.option("-a", "--authdb", help="database to use for authentication")
+@click.option("-port", "--port", help="Specify on what port to listen for the mongod")
+@click.option("-h", "--host", help="Specify the host for the mongo database.")
+@click.option("--uri")
+@click.option(
+    "-c",
+    "--config",
+    type=click.Path(exists=True),
+    help="Specify the path to a config file with database info.",
+)
+@click.version_option(__version__)
+@click.pass_context
+def tabulate_causative_panel_rank(
+    context, mongodb, username, password, authdb, host, port, loglevel, config, uri
+):
+    """scout: manage interactions with a scout instance."""
```

Comment:
Perhaps you could expand here and describe what is the expected result from this script!